__tests__/app.test.js

```diff
@@ -85,3 +85,48 @@ describe("GET /api/articles/:article_id", () => {
     });
 })
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200 sends an array of comments to the client that matched the article_id", () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toBeSortedBy( 'created_at', {descending: true})
+        expect(body.comments.length).toBe(2);
+        body.comments.forEach((comment) => {
```

Comment:
you're designing a lot of behaviours in one test. We ideally want one test per behaviour. I'd break the order test into different test, so if you break the sorting, you know exactly what you broke.